packages/manager/apps/dedicated/client/app/dedicated/server/install/image/components/config-drive/controller.js

```diff
@@ -47,5 +48,20 @@ export default class DedicatedServerInstallImageConfigDriveCtrl {
     this.model.configdrive.metadata.splice(index, 1);
   }
 
+  trackClick(name) {
```

Comment:
no need to create a trackClick function you can directly call atInternet.trackClick where it's needed
(you only call it once btw)